test/functional/is-browser-relevant.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function isBrowserRelevant (testName, browserAlias) {

```

Comment:
It's just a single function that used in one place. It's definetly belongs to the place there it was called.